<?xml version="1.0" encoding="UTF-8"?>
<!-- Launch control from teleop, sending high-level command messages
-->
<launch>
  <node pkg="rovr_control" type="robot_to_mc_node" name="robot_to_mc_node" output="screen" respawn="true">
    <param name="frequency" value="20.0" type="double"/>
    <param name="dst_addr" value="192.168.1.10" type="str"/>
    <param name="dst_port" value="5554" type="int"/>
    <param name="src_port" value="5519" type="int"/>
    <param name="isTeleopCtrl" value="true" type="bool"/>
  </node>

  <include file="highlevel_control.launch" />

</launch>
